atmo/apps.py

```diff
@@ -79,6 +79,9 @@ def ready(self):
         # https://github.com/mozilla/sugardough/issues/38
         session_csrf.monkeypatch()
 
+        # Import job signals so they are registered.
+        import atmo.jobs.signals  # noqa
```

Comment:
IOW, don't use the decorator syntax but actually call `.connect` here? Sure, sounds better.